content/kotlin/concepts/arrays/map/map.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.map()'
+Description: 'map() is a Kotlin method that transforms elements in a collection.'
+Subjects: 
+  - 'Code Foundations'
+  - 'Mobile Development'
+Tags: 
+  - 'Android'
+  - 'Arrays'
+  - 'Map'
+  - 'Kotlin'
+CatalogContent: 
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+**map()** is a Kotlin method that transforms elements in a collection (Array, List, or Set) using the provided logic. It does not modify the original collection, which can be useful when data permanence is required.
+
+
+## Syntax
+collection.map { element -> transformation logic }
```

Comment:
```suggestion
## Syntax
collection.map { element -> transformation logic }
```
The syntax should be always written inside the `pseudo` block.

Reply:
This change appears identical to the original, could you please clarify? Are you saying I need to put this inside a pseudo block?

Reply:
@SaviDahegaonkar I realized you might not have noticed the comment because I didn't tag you.

Reply:
yes I mean the same